src/helpers/index.ts

```diff
@@ -38,3 +40,11 @@ export const createFaviconNode = (): void => {
 export const isBrowserPreferenceModeDark = (): boolean => {
   return window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches
 }
+
+export const getChildrenWithNewProps = (children: any, props: any): ReactNode => { // eslint-disable-line
```

Comment:
Please please please avoid the combo `any` and `// eslint-disable-line` 😃 Ping me first next time I will help you 👍

Reply:
i couldn't resolve type of children and props here. feel free to suggest some types here.

Reply:
I will show you again monday how you can easily find types coming from an external library 👍

Reply:
but this is not just for an external library. I need to copy and inject all possible props of all possible types to the children components here. that's why i thought `any` is acceptable in this case.

Reply:
Guess that this discussion is still not resolved 😅

Reply:
that function is no longer useful. so i removed it